src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -24,11 +25,16 @@ trait ThurloeService extends HttpService {
 
   val getRoute = path(ThurloePrefix / Segment / Segment) { (userId, key) =>
     get {
-      onComplete(dataAccess.lookup(userId, key)) {
-        case Success(keyValuePairWithId) =>
+      val query: Future[Seq[UserKeyValuePair]] = if ("*".equals(userId)) { dataAccess.lookupByKey(key) } else { dataAccess.lookup(userId, key) map { Seq(_) } }
```

Comment:
using a wildcard seems kind of icky. I'd rather see a separate end point but I see how that might be difficult given the current structure